semperfiwebdesign/all-in-one-seo-pack
# Add support for Google Tag Manager

Issue #2983

## Proposed changes

This is the code that has to go in free for this feature.

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist


## Testing instructions

Build this with the changes from the Pro PR and make sure that I didn't break anything from moving this part of the code to the free repo.
